src/pages/other-services/resources-and-toolkits/covid-19-kit-for-parents.tsx

```diff
@@ -0,0 +1,103 @@
+import Link from "next/link";
+
+import { List } from "@/components/Services/list";
+
+export default function ResourcesAndToolkits() {
+  const covidBookletList = (
+    <List
+      items={[
+        "families and organisations with practical tips, activities and links to resources that are focused on women and children and young people’s health and wellbeing.",
+        "offers guidance, support and information for women whose families are in isolation."
+      ]}
+    />
+  );
+
+  const body = (
+    <>
+      <div className="flex justify-center bg-[--starick-white] px-6 py-2 md:px-32">
+        <div className="w-fit">
+          <br></br>
```

Comment:
Nitpick but can be a self-closing tag instead (there's a few more instances of this below as well)